packages/components/package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "@terra-ui-packages/components",
+  "version": "0.0.1",
+  "private": true,
+  "scripts": {
+    "build": "tsc && tsc -p tsconfig.cjs.json",
+    "test": "jest"
+  },
+  "type": "module",
+  "module": "./lib/es/index.js",
+  "main": "./lib/cjs/index.js",
+  "exports": {
+    ".": {
+      "import": "./lib/es/index.js",
+      "require": "./lib/cjs/index.js"
+    }
+  },
+  "files": [
+    "lib/cjs/**",
+    "lib/es/**"
+  ],
+  "dependencies": {
+    "lodash": "^4.17.21",
+    "react-hyperscript-helpers": "^2.0.0"
+  },
+  "devDependencies": {
+    "@terra-ui-packages/test-utils": "*",
+    "@types/jest": "^28.1.8",
+    "@types/lodash": "^4.14.184",
+    "@types/react": "^18.2.15",
+    "jest": "^27.4.3",
+    "react": "18.2.0",
+    "typescript": "^4.9.5"
+  },
+  "peerDependencies": {
+    "react": "18.2.0"
```

Comment:
React must be specified as a peer dependency and not a dependency to avoid packages pulling multiple versions of React into the Terra UI bundle.

Reply:
I might move this excellent comment into the file above.